### ISSUE TITLE
Proc macro host zombies

### ISSUE THREAD

**Issue by bjorn3** [rust-analyzer/rust-analyzer#5559]
```
bjorn    13324  0.0  0.0      0     0 ?        Z    19:01   0:00 [rust-analyzer-l] <defunct>
bjorn    13731  0.0  0.0      0     0 ?        Z    19:02   0:00 [rust-analyzer-l] <defunct>
bjorn    13913  0.0  0.0      0     0 ?        Z    19:02   0:00 [rust-analyzer-l] <defunct>
bjorn    14902  0.0  0.0      0     0 ?        Z    19:03   0:00 [rust-analyzer-l] <defunct>
bjorn    16786  0.0  0.0      0     0 ?        Z    19:05   0:00 [rust-analyzer-l] <defunct>
bjorn    17081  0.0  0.0      0     0 ?        Z    19:06   0:00 [rust-analyzer-l] <defunct>
bjorn    17880  0.0  0.0  20592   596 ?        S    19:06   0:00 /home/bjorn/.var/app/com.visualstudio.code/config/Code/User/globalStorage/matklad.rust-analyzer/rust-analyzer-linux proc-macro
bjorn    20285  5.8 31.5 3031284 1244640 ?     Sl   17:17   8:42 /home/bjorn/.var/app/com.visualstudio.code/config/Code/User/globalStorage/matklad.rust-analyzer/rust-analyzer-linux
bjorn    20371  0.0  0.0      0     0 ?        Z    17:17   0:00 [rust-analyzer-l] <defunct>
bjorn    22255  0.0  0.0      0     0 ?        Z    17:38   0:00 [rust-analyzer-l] <defunct>
bjorn    22337  0.0  0.0      0     0 ?        Z    17:38   0:00 [rust-analyzer-l] <defunct>
bjorn    22389  0.0  0.0   6228  2372 pts/0    S+   19:46   0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn rust-analyzer
bjorn    24190  0.0  0.0      0     0 ?        Z    17:39   0:00 [rust-analyzer-l] <defunct>
```

**Comment by ruabmbua** [comment]
Hm I had some zombie cargo and rustc processes a while ago.
Unfortunately I ignored it, and did not report it.

**Comment by 0xpr03** [comment]
I've got some more here
![image](https://user-images.githubusercontent.com/1280209/96168611-47ed3f00-0f21-11eb-81b4-d27de4a6c1d1.png)

**Comment by lnicola** [comment]
Presumably fixed in #7398.
